### PR TITLE
Pimo tutorials/02 advanced ii

### PR DESCRIPTION
**Pull request by jpcbertoldo**
## 📝 Description

add aupimo notebook advanced ii (pimo curve and integration bounds)

## ✨ Changes

Select what type of change your PR is:

- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🔨 Refactor (non-breaking change which refactors the code base)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] 📚 Documentation update
- [ ] 🔒 Security update

## ✅ Checklist

Before you submit your pull request, please make sure you have completed the following steps:

- [ ] 📋 I have summarized my changes in the [CHANGELOG](https://github.com/openvinotoolkit/anomalib/blob/main/CHANGELOG.md) and followed the guidelines for my type of change (skip for minor changes, documentation updates, and test enhancements).
- [ ] 📚 I have made the necessary updates to the documentation (if applicable).
- [ ] 🧪 I have written tests that support my changes and prove that my fix is effective or my feature works (if applicable).

For more information about code review checklists, see the [Code Review Checklist](https://github.com/openvinotoolkit/anomalib/blob/main/docs/source/markdown/guides/developer/code_review_checklist.md).

### CODE REVIEW

**Comment by jpcbertoldo** [comment]
to be opened after https://github.com/openvinotoolkit/anomalib/pull/2336 is merged

File: notebooks/700_metrics/701c_aupimo_advanced_ii.ipynb [hunk 1]
```diff
@@ -0,0 +1,936 @@
+{
```
Comment (ashwinvaidya17) [inline notebooks/700_metrics/701c_aupimo_advanced_ii.ipynb:1]: <p>I think the 100_datamodules link is broken</p><p>Also, are you sure you want to link the 701a_aupimo to the raw notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/2347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Review by ashwinvaidya17 — COMMENTED** [review]
Super minor comments but apart from these, I am okay with the PR

File: notebooks/700_metrics/701c_aupimo_advanced_ii.ipynb (hunk 1 above)
Comment (jpcbertoldo) [reply notebooks/700_metrics/701c_aupimo_advanced_ii.ipynb:1]: fixed both, also in previous notebooks

**Review by ashwinvaidya17 — APPROVED** [review]

**Comment by jpcbertoldo** [comment]
@ashwinvaidya17  i corrected what you said here https://github.com/openvinotoolkit/anomalib/pull/2336#pullrequestreview-2356476782

can we merge this one?

**Pull request merged**